Resources/Changelog/WhiteChangelog.yml

```
@@ -2936,3 +2936,10 @@ Entries:
   id: 317
   time: '2025-05-25T05:12:22.0000000+00:00'
   url: https://github.com/WWhiteDreamProject/wwdpublic/pull/515
+- author: Spatison
+  changes:
+    - type: Fix
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)